README.md

```diff
@@ -1,261 +1,79 @@
-<h1 align="center">
-    <br>
-    Av1an
-    </br>
-</h1>
+# Av1an
 
-<h2 align="center">A cross-platform framework to streamline video encoding</h2>
+[![Discord server](https://discordapp.com/api/guilds/696849974230515794/embed.png)](https://discord.gg/Ar8MvJh)
+[![CI tests](https://github.com/master-of-zen/Av1an/actions/workflows/tests.yml/badge.svg)](https://github.com/master-of-zen/Av1an/actions/workflows/tests.yml)
+[![](https://img.shields.io/crates/v/av1an.svg)](https://crates.io/crates/av1an)
 
-![alt text](https://cdn.discordapp.com/attachments/804148977347330048/928879953825640458/av1an_preview.jpg)
+Av1an is a video encoding framework for modern encoders. It can increase your encoding efficiency by running multiple encoder processes in parallel. This can improve CPU usage and increases the speed of some AV1 encoders dramatically.
```

Comment:
I might emphasize here that the reason it increases speed so much in some cases is because of poor multi-threading in the encoders themselves. I think I would also change "some AV1 encoders" to maybe just "some encoders", since libvpx also gets a similarly large speedup.

Reply:
Good catch, I didn't know about vpxenc. Let me know if you like the new wording.